src/vma/dev/net_device_val.cpp

```diff
@@ -829,11 +834,15 @@ void net_device_val_ib::configure(struct ifaddrs* ifa, struct rdma_cm_id* cma_id
 
 	create_br_address(m_name.c_str());
 
-	g_p_neigh_table_mgr->unregister_observer(neigh_key(ip_address((in_addr_t)(inet_addr(BROADCAST_IP))), this), this);
+	if (1 != inet_pton(AF_INET, BROADCAST_IP, &in)) {
```

Comment:
Aren't you looking for succeess return value? Shouldn't it be "equal to 1" rather than "not equal to 1"?